R/check-class.R

```diff
@@ -102,17 +111,38 @@ check_update <- function(self, private) {
 
 check_web <- function(self, private, which) {
 
+  ids <- select_ids(which = which, self = self, 
+                    private = private)
+
+  urls <- paste0(sub("/api$", "/status/", baseurl()), ids)
+  
+  lapply(urls, browseURL)
+  invisible(self)
+}
+
+check_urls <- function(self, private, which) {
+  
+  ids <- select_ids(which = which, self = self, 
+                    private = private)
+  
+  data.frame(html = paste0(sub("/api$", "/status/", baseurl()), ids),
+             text = paste0(sub("/api$", "/status/original/", baseurl()), 
+                           ids),
+             artifacts = paste0("https://artifacts.r-hub.io/", ids),
+             stringsAsFactors = FALSE)
```

Comment:
Actually, one more thing. I think `group` and `id` should be also in the data frame, no? If it is already a data frame....

Reply:
only group_id and check_id or also other stuff like platform?

Reply:
Yeah. But then we could probably call it sg else, and and have separate methods  that only return the log url, the artifact url, etc.

Reply:
Anyway, I think I'll merge this now, and will improve after....